app/src/main/java/cz/martykan/forecastie/activities/MainActivity.java

```diff
@@ -900,6 +959,42 @@ protected void onPostExecute(TaskOutput output) {
         }
     }
 
+    class TodayUVITask extends GenericRequestTask {
+        public TodayUVITask(Context context, MainActivity activity, ProgressDialog progressDialog) {
+            super(context, activity, progressDialog);
+        }
+
+        @Override
+        protected void onPreExecute() {
+            loading = 0;
+            super.onPreExecute();
+        }
+
+        @Override
+        protected void onPostExecute(TaskOutput output) {
```

Comment:
Don't think you need to override this method if you're only calling the super's implementation.